apps/docs/content/components/Heading.mdx

```diff
@@ -104,15 +107,85 @@ Use the `weight` prop to adjust the default visual appearance if required.
 </>
 ```
 
+### Width
+
+The recommended text width is automatically applied to Headings based on their position on the type scale.
+
+Use the `width` prop to override the default visual appearance if required.
+
+```jsx live
+<>
+  <Heading as="h3" width="condensed">
+    condensed
+  </Heading>
+  <Heading as="h3" width="normal">
+    normal
+  </Heading>
+  <Heading as="h3" width="expanded">
+    expanded
+  </Heading>
+
+  {/* Responsive */}
+  <br />
+  <br />
+  <Heading
+    as="h4"
+    width={{
+      narrow: 'condensed',
+      regular: 'normal',
+      wide: 'expanded',
+    }}
+  >
+    Responsive
+  </Heading>
+</>
+```
+
+### Letter spacing
+
+The recommended character letter spacing is automatically applied to Headings based on their position on the type scale.
+
+Use the `letterSpacing` prop to override the default settings if required.
+
+```jsx live
+<>
+  <Heading as="h3" letterSpacing="condensed">
```

Comment:
```suggestion
  <Heading as="h3" letterSpacing="tight">
```

Same here. This should be changed, unless we stick with `condensed`

Reply:
n/a as we switched to `condensed` 👍